tritonbench/components/do_bench/run.py

```diff
@@ -185,6 +185,14 @@ def _do_bench_profiler(
     Returns:
```

Comment:
After this PR, the parameter `warmup` will not be used by this function. Can you help remove it?

Reply:
let's keep it to match the API of `_do_bench_inductor` and `triton.testing.do_bench`